code/modules/surgery/organs/heart.dm

```diff
@@ -286,3 +286,9 @@
 		owner.heal_overall_damage(15, 15, 0, BODYPART_ORGANIC)
 		if(owner.reagents.get_reagent_amount(/datum/reagent/medicine/ephedrine) < 20)
 			owner.reagents.add_reagent(/datum/reagent/medicine/ephedrine, 10)
+
+/obj/item/organ/heart/freedom/nanomachine
+	name = "Nanomachine heart"
+	desc = "An EMP proof heart that provides an additional boost to nanites and emergency healing"
+	var/nanite_boost = 2
```

Comment:
I think that just adding a variable that isn't used anywhere wouldn't make nanites have double regen

Reply:
lol the variable is already used in the nanite heart https://github.com/yogstation13/Yogstation/blob/master/yogstation/code/modules/surgery/organs/heart.dm

Reply:
It doesn't work like this, the variable itself doesn't do anything if you don't use it anywhere. The nanite hear has some code that makes it generate nanites. Your item doesn't.